.github/workflows/test-cygwin.yml

```diff
@@ -67,7 +67,7 @@ jobs:
             python3${{ matrix.python-minor-version }}-numpy
             python3${{ matrix.python-minor-version }}-sip
             python3${{ matrix.python-minor-version }}-tkinter
-            qt5-devel-tools
+            qt6-devel-tools
```

Comment:
```suggestion
```

qt6-devel-tools isn't found in Python [3.8](https://github.com/python-pillow/Pillow/actions/runs/4629282314/jobs/8189344115#step:4:74) or [3.9](https://github.com/python-pillow/Pillow/actions/runs/4629282314/jobs/8189344329#step:4:74)

Reply:
Looks like Cygwin doesn't provide any Qt6 packages: https://www.cygwin.com/packages/package_list.html#q